index.html

```diff
@@ -3128,6 +3128,21 @@ <h3>Canonicalization Method Correctness</h3>
         </p>
       </section>
 
+      <section>
+        <h3>Other Security Considerations</h3>
+
+        <p>
+Since the technology described by this specification to secure documents is
+generalized in nature, the security implications of its use might not be
+immediately apparent to readers. In order to understand the sort of security
+concerns one might need to consider in a complete software system, implementers
+are urged to read about how this technology is utilized in the
+<a>verifiable credentials</a> ecosystem [[?VC-DATA-MODEL-2.0]]; see the section
+on <a data-cite="VC-DATA-MODEL-2.0#security-considerations">
+Verifiable Credential Security Considerations</a> for more information.
```

Comment:
```suggestion
Since the technology to secure documents described by this specification is
generalized in nature, the security implications of its use might not be
immediately apparent to readers. To understand the sort of security
concerns one might need to consider in a complete software system, implementers
are urged to read about how this technology is used in the
<a>verifiable credentials</a> ecosystem [[?VC-DATA-MODEL-2.0]]; see the section
on <a data-cite="VC-DATA-MODEL-2.0#security-considerations">
Verifiable Credential Security Considerations</a> for more information.
```